tests/cpp-tests/Classes/controller.cpp

```diff
@@ -56,11 +56,11 @@ class RootTests : public TestList
 
 
         addTest("Box2D - Basic", []() { return new (std::nothrow) Box2DTests(); });
-#if defined(CC_PLATFORM_PC)
+#if (CC_TARGET_PLATFORM == CC_PLATFORM_WIN32)
```

Comment:
The box2d testbed can't be run on linux or mac ?

Reply:
I have No system to chack it.
IT should

Reply:
Chipmunk testbed also

Reply:
ok, I will check them

Reply:
The prob is:
I will add Mouse and Keyboard Support later and after all i will move Dome Files into extension

Reply:
I belive some thirt Party stuff have to be adapt dir Linux/Mac

Reply:
For cross-platform adaption, you can refer to https://github.com/halx99/adxe/commit/3d7251e7e3864713dd2184da62779bcdfcd9be86

Reply:
You will pull all changes to the dev branch?
Or have I to somthing before?

Reply:
If the changes no conflicit with your branch `Box2D-TestBed`, I can create a pr with my branch

Reply:
Thanks.

Reply:
Checked, no confilcit, and I create a pull-request: https://github.com/adxeproject/adxe/pull/420

Reply:
Perfect. Thanks!
I will working again after all checkins.

Reply:
The pr 420 was merged, can I merge this pr before you continue working?

Reply:
That what I mean.
I will wait.

Reply:
Merged, you can create a new branch from upstream:dev and continue working

Reply:
> 
> 
> The box2d testbed can't be run on linux or mac ?

Have you tested it on mac?

I see on the last upload on controller.cpp:   (So only Windows can the the menu "Box2D - TestBed")
```
#if (CC_TARGET_PLATFORM == CC_PLATFORM_WIN32)
        addTest("Box2D - TestBed", []() { return new (std::nothrow) Box2DTestBedTests(); });
#endif
```

Reply:
I run the chipmunk-testbed on mac, works fine.

Reply:
Box2D - Testbed is crashing on linux